Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -11,6 +11,10 @@ internal protocol AddNewCardViewControllerDelegate: AnyObject {
     didSucceedWithMessage message: String
   )
   func addNewCardViewControllerDismissed(_ viewController: AddNewCardViewController)
+  func addNewCardViewController(
```

Comment:
This can be:

```swift
func addNewCardViewController(
  _ viewController: AddNewCardViewController,
  didAdd newCard: GraphUserCreditCard.CreditCard
)
```